frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts

```diff
@@ -106,6 +133,13 @@ export class SendAlarmGroupInterfaceComponent implements OnDestroy {
                         MapCoordinates.create(0, 0)
                     );
 
+                    const vehicleTargetTransferPointId =
+                        vehicleIndex <
+                            firstAlarmGroupVehiclesToOtherLocationCount &&
+                        this.firstVehiclesToOtherLocationTransferPointId
+                            ? this.firstVehiclesToOtherLocationTransferPointId
+                            : this.targetTransferPointId!;
```

Comment:
Using the index does not work if the vehicles are not ordered by their arrival time with shortest arrival time first. This could occur frequently as the vehicles in an alarm group are ordered by the order they have been added and there is no way to reorder them, thus vehicles that might have been forgotten to add will be the last ones although they might arrive first.

Reply:
Done in d81b5f0823d86c3041b6c8f305072b1fcd177014